mne/time_frequency/tests/test_psd.py

```diff
@@ -13,102 +15,193 @@
-    raw.info['bads'] += ['MEG 2443']  # bads
+    # Multitaper
+    picks_mt = picks[:2]
+    kws_mt = dict(tmin=tmin, tmax=tmax, fmin=fmin, fmax=fmax, low_bias=True)
+    psds, freqs = psd_multitaper(raw, proj=False, picks=picks_mt, **kws_mt)
+    psds_proj, freqs_proj = psd_multitaper(raw, proj=True,
+                                           picks=picks_mt, **kws_mt)
 
-    # picks MEG gradiometers
-    picks = pick_types(raw.info, meg='grad', eeg=False, eog=True,
-                       stim=False, include=include, exclude='bads')
+    assert_array_almost_equal(psds, psds_proj)
+    assert_true(psds.shape == (len(picks_mt), len(freqs)))
+    assert_true(np.sum(freqs < 0) == 0)
+    assert_true(np.sum(psds < 0) == 0)
 
+    # -- Epochs --
     events = read_events(event_fname)
-
+    tmin, tmax, event_id = -0.5, 0.5, 1
     epochs = Epochs(raw, events[:10], event_id, tmin, tmax, picks=picks,
                     baseline=(None, 0),
                     reject=dict(grad=4000e-13, eog=150e-6), proj=False,
                     preload=True)
 
     tmin_full, tmax_full = -1, 1
-    epochs_full = Epochs(raw, events[:10], event_id, tmax=tmax_full,
-                         tmin=tmin_full, picks=picks,
-                         baseline=(None, 0),
+    epochs_full = Epochs(raw, events[:10], event_id, tmin_full, tmax_full,
+                         picks=picks, baseline=(None, 0),
                          reject=dict(grad=4000e-13, eog=150e-6), proj=False,
                          preload=True)
 
-    picks = pick_types(epochs.info, meg='grad', eeg=False, eog=True,
-                       stim=False, include=include, exclude='bads')
-    psds, freqs = compute_epochs_psd(epochs[:1], fmin=2, fmax=300,
-                                     n_fft=n_fft, picks=picks)
+    # - Welch -
+    picks_wel = pick_types(epochs.info, meg=True, eeg=True, ref_meg=False,
+                           exclude='bads')
+    kws_welch = dict(fmin=2, fmax=300, n_fft=512, picks=picks_wel)
+    psds, freqs = psd_welch(epochs[:1], proj=False, **kws_welch)
+    psds_full, freqs_full = psd_welch(epochs_full[:1], proj=False,
+                                      tmin=tmin, tmax=tmax, **kws_welch)
+    psds_proj, freqs_proj = psd_welch(epochs[:1], proj=True, **kws_welch)
 
-    psds_t, freqs_t = compute_epochs_psd(epochs_full[:1], fmin=2, fmax=300,
-                                         tmin=tmin, tmax=tmax,
-                                         n_fft=n_fft, picks=picks)
     # this one will fail if you add for example 0.1 to tmin
-    assert_array_almost_equal(psds, psds_t, 27)
+    assert_array_almost_equal(psds, psds_full, 27)
+    assert_array_almost_equal(psds, psds_proj, 27)
+
+    assert_true(psds.shape == (1, len(picks_wel), len(freqs)))
+    assert_true(np.sum(freqs < 0) == 0)
+    assert_true(np.sum(psds < 0) == 0)
 
-    psds_proj, _ = compute_epochs_psd(epochs[:1].apply_proj(), fmin=2,
-                                      fmax=300, n_fft=n_fft, picks=picks)
```

Comment:
compute_epochs_psd is tested in the to-be-deprecated other PSD test. I was keeping them separate but can combine if that's better